tests/unit/s2n_certificate_extensions_test.c

```diff
@@ -59,7 +59,7 @@ const char tls13_cert_invalid_ext_hex[] =
 /* with single extension sent */
 /* server can send empty status request extension from https://tools.ietf.org/html/rfc8446#section-4.4.2.1 */
 const char tls13_cert_single_ext_hex[] =
-    "000001b90001b03082" /* without 0b0001b9 header */
+    "000001ba0001b03082" /* without 0b0001b9 header */
```

Comment:
what changed here?

Reply:
The size of the total message changed from 01b9 -> 01ba because I added one more byte.